static/src/javascripts/projects/common/modules/experiments/tests/remote-epic-variants.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+
+import { setEpic } from "common/modules/commercial/contributions-service";
+import config from 'lib/config';
+
+const id = 'RemoteEpicVariants';
+
+const remoteVariant: Variant = {
+    id: 'remote',
+    test: () => setEpic(id),
+    canRun: () => true,
+};
+
+export const remoteEpicVariants: Runnable<ABTest> = {
+    id,
+    start: '2020-05-01',
+    expiry: '2020-07-01',
+    author: "Nicolas Long",
+    description: "Pseudo-test to use remote service for % of contribution epics. Expected to run as highest priority test; the canRun will then narrow the audience.",
+    audience: 1,
+    audienceOffset: 0,
+    successMeasure: "Revenue/impressions equivalent to local variants",
+    audienceCriteria: "All",
+    variants: [remoteVariant],
+    canRun: () =>
```

Comment:
You are missing the showForSensitive flag, which is now required (https://github.com/guardian/frontend/pull/22548)

Reply:
Thanks, yeah looks like a rebase is in order.